### ISSUE TITLE
How to update LUIS dispatched model when child LUIS model changes ?

### ISSUE THREAD

**Issue by rjgmail88** [Microsoft/botbuilder-js#273]
I implemented LUIS dispatcher using [1] with my 1 LUIS model & 1 QnA. Now, I am in a situation where I plan to have more QnA and new LUIS model connected to same bot. 

How can I use dispatcher to make new QnA and new LUIS under the same umbrella ?
Dispatcher tool just combines utterances and intents to new model. So is there any way to add new QnA and new LUIS model or Intent as a part of dispatcher ? 

[1][https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-dispatch?view=azure-bot-service-4.0&branch=master&tabs=csaddref%2Ccsbotconfig](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-dispatch?view=azure-bot-service-4.0&branch=master&tabs=csaddref%2Ccsbotconfig)

**Comment by Stevenic** [comment]
@CezaryMarcjan can you assist?

**Comment by CezaryMarcjan** [comment]
Yes, you can add additional components (luis or qna) by running additional "dispatch add ..." commands to your dispatcher for each new component. Dispatcher supports multiple LUIS apps and multiple QnA KBs.
